converter/src/test/java/gov/cms/qpp/conversion/decode/QrdaDecoderEngineTest.java

```diff
@@ -327,13 +320,12 @@ void testDontPruneInsignificantChildrenSignificantGrandChildrenWhenDefaults() {
 			QrdaDecoderEngine objectUnderTest = new QrdaDecoderEngine(context);
 			Node decodedNodes = objectUnderTest.decode(rootElement);
 
-			assertNodeCount(decodedNodes, 1, 0, 0);
+			assertNodeCount(decodedNodes, 0, 0, 0);
```

Comment:
I believe that this test is now the same as below with the removal of defaults.  @clydet can confirm.